src/VisualStudio/Core/Def/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -225,6 +226,10 @@ public async Task InitializeUIAffinitizedServicesAsync(IAsyncServiceProvider asy
 
             Logger.Log(FunctionId.Run_Environment,
                 KeyValueLogMessage.Create(m => m["Version"] = FileVersionInfo.GetVersionInfo(typeof(VisualStudioWorkspace).Assembly.Location).FileVersion));
+
+            // Initialize task-list in BG.
+            var taskListService = this.Services.SolutionServices.ExportProvider.GetExports<VisualStudioTaskListService>().Single().Value;
+            taskListService.Start(this);
```

Comment:
💭 Where is this normally initialized? Does it need to be done so early in startup?

Reply:
this is normally initialized when the workspace calls EnsureEventListeners.  This happens the moment teh workspace assigns itself any solution (including hte empty one).